packages/commonwealth/client/scripts/views/components/user/fullUser.tsx

```diff
@@ -44,13 +44,12 @@ export const FullUser = ({
   const loggedInUser = useUserStore();
   const [isModalOpen, setIsModalOpen] = useState(false);
 
-  const { data: userCommunity, isLoading: isLoadingUserCommunity } =
-    useGetCommunityByIdQuery({
-      id: userCommunityId || '',
-      enabled: !!userCommunityId,
-    });
+  const { data: userCommunity } = useGetCommunityByIdQuery({
+    id: userCommunityId || '',
+    enabled: !!userCommunityId,
+  });
 
-  if (showSkeleton || isLoadingUserCommunity) {
+  if (showSkeleton) {
```

Comment:
this would break in many cases. If `userCommunityId` is `undefined` or `''`, then the component passing that prop to this component, isn't passing the prop correctly. Lets fix it there.

Reply:
> this would break in many cases. If `userCommunityId` is `undefined` or `''`, then the component passing that prop to this component, isn't passing the prop correctly. Lets fix it there.

@mzparacha  issue is that for me locally is working fine not able to reproduce the bug .even i have latest dump